### ISSUE TITLE
How do I validate that enable tracing?

### ISSUE THREAD

**Issue by smeubank** [getsentry/sentry-docs#10503]
### Core or SDK?

Platform/SDK

### Which part? Which one?

All but let's start with react based Next.js

### Description

Our docs, at least for setting up errors, have followed a simple format at their core: 1. install 2. configure 3. validate

This is establishes a pretty straigh forward path to set it up, users generally know what errors, are and an example to get you going using Sentry. For tracing we do not follow such a pattern exactly.

Starting from the [Next.js/Set Up Tracing](https://docs.sentry.io/platforms/javascript/guides/nextjs/tracing/) docs, i first presented with a pagraph on "tracing . . . performance . . . measuring metrics . . . displaying impact of errors across multiple systems"

What? That is so far ahead of where the developer is at this stage, and even is some pretty broad strokes claims I am not sure of. So what exactly should I expect when I turn this on? What will a "trace" be in Sentry for my Next.js application?

### Suggested Solution

1. remove|replace both of those notes which appear before configure
2. move distributed tracing upsell lower
3. add bullet points > this is SOME of what you should see that will create traces automatically for Next.js
   - Page loads and navigations
   - Functions
     - Server Components (Layout and Page)
     - Page.generateMetadata (i think this is too specific)   
   - Middleware 

![image](https://github.com/getsentry/sentry-docs/assets/47563310/50a819ba-3a95-4cd8-90db-97d6386525c4)

**Comment by stephanie-anderson** [comment]
I fully agree, thanks for taking the time to improve this! What I was thinking was, to also remove the `traces_sampler` instructions from this page, and reference it as advanced use case. This would further simplify the instructions

**Comment by smeubank** [comment]
> to also remove the `traces_sampler` instructions from this page, and reference it as advanced use case.

i agree, i want to think for the other PR to change it to tell the user to verify not based on sampling at all, but to go check for their page loads etc.
